.github/workflows/android-build.yml

```diff
@@ -42,7 +42,7 @@ jobs:
           key: ${{ runner.os }}-gradle-${{ hashFiles('**/*.gradle*', '**/gradle-wrapper.properties') }}
           restore-keys: |
             ${{ runner.os }}-gradle-
-      - run: command -v uniffi-bindgen || cargo install uniffi_bindgen --version 0.18.0
+      - run: cargo install uniffi_bindgen --version 0.19.1
```

Comment:
will it be always reinstalling now?

Reply:
No. I try it on local, it only displayed a message as below:
```
% cargo install uniffi_bindgen --version 0.19.1
Ignored package `uniffi_bindgen v0.19.1` is already installed, use --force to override
```